client/my-sites/plan-features/index.jsx

```diff
@@ -209,10 +205,7 @@ class PlanFeatures extends Component {
 			} = properties;
 			let { rawPrice, discountPrice } = properties;
```

Comment:
Now that we're not reassigning `rawPrice` and `discountPrice`, we can change that `let` to `const`.